common/components/Header/components/GasPriceDropdown.scss

```diff
@@ -0,0 +1,17 @@
+@import "../../../../common/sass/variables";
```

Comment:
You can just to `@import "common/sass/variables";`, it looks from the top of the repo.